WeakAuras/WeakAuras.lua

```diff
@@ -4539,10 +4546,9 @@ local function ApplyStateToRegion(id, region, state)
   end
 
   WeakAuras.UpdateMouseoverTooltip(region);
-
   region:Expand();
-  if (controlChidren) then
-    WeakAuras.ControlChildren(region.id);
+  if reindex and parent and parent.ActivateChild then
```

Comment:
Don't introduce this code that checks if the duration/expirationTime has changed. It's up to the triggers to set the resort property, if they touch expirationTime and/or index. 

Also SetDurationInfo is mostly going away anyway in my branch.

Reply:
Does every trigger correctly set resort when necessary currently?

Reply:
Ah, it looks like they do. My mistake.